core/storage/base_model/gae_models_test.py

```diff
@@ -335,10 +335,9 @@ def mock_put(self):
             if self.last_updated is None:
                 self.last_updated = datetime.datetime.utcnow()
 
-            # We have not used BaseHumanMaintainedModel.put() because it is
-            # not implemented and throws error rather than saving the changes.
-            # We have used BaseModel.put() because we wanted the
-            # changes made above to be saved.
+            # We are using BaseModel.put() to save the changes to the datastore
+            # since the put() method which TestBaseHumanMaintainedModel class
+            # inherits from BaseHumanMaintainedModel raises NotImplementedError.
```

Comment:
Solves [this](https://github.com/oppia/oppia/pull/13413#discussion_r683739651) comment as per earlier PR.

Reply:
"....raises NotImplementedError, and we do actually want to save the changes in this case."

Reply:
Done